### DISCUSSION TITLE
Illegible parts of a name

### DISCUSSION THREAD

**Discussion by hartenthaler** [fisharebest/gedcom-name#19] [General]
For illegible parts of a name from a source, the GEDCOM standard 5.5.1 requires the use of an ellipsis. A character with three dots should be used: '…'. So if parts of the surname cannot be recognized in a church register entry, one should record recognized parts and mark gaps:

`1 NAME Johannes /Bal…grien/`

In GEDCOM 7.0 I didn't find that recommendation. Should it be there or is it obsolete?

**Comment by ungeahnt** [comment]
It's definitely neccessary to enter and mark illegible parts. The marker (= ellipsis, dots or wathever) has to be decided later.

**Reply by hartenthaler** [reply]
ok, but why not use what was already defined in GEDCOM 5.5.1? This has the chance that all GEDCOM files using the character '…' are still valid.

In your example in the duplicated issue #22 you used 
> 1 NAME Mar(...) /(A...)men(dorf)/

For me, the "(" and ")" are not necessary, the character '…' is enough in order to indicate that there are one or more characters illegible. But what is the meaning of (A...) and (dorf)? I would write your example in the following way:
> 1 NAME Mar… /A…mendorf/

**Reply by ungeahnt** [reply]
> in the duplicated issue #22 you used

You were faster ... one month :)

>  I would write your example in the following way

In the end, I don't care how this is solved in GEDCOM. The main thing is that it is possible somehow.

**Reply by ungeahnt** [reply]
btw: with the parentheses I group ambiguous data. That would be another point, I think.
